docs/zh/latest/README.md

```diff
@@ -39,30 +39,45 @@ Apache APISIX 的技术架构如下图所示：
 ## 社区
 
 - 邮件列表 - 发送任意内容到 dev-subscribe@apisix.apache.org 后，根据回复以订阅邮件列表。
+
```

Comment:
Might need to tweak IDE formatting tools or just turn it off temporarily, so unrelated changes like this won't be committed